build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
```

Comment:
nit: thinking out loud, should we add the ability to trace the progress of this script by setting the `xtrace` option if e.g. a "TRACE" env var is set too?

Reply:
I had made this a /bin/sh script since I didn't think that FreeBSD shipped with bash in the default install, but it looks like they do, it's just not located at /bin/bash but at /usr/local/bin/bash instead. I can probably convert this to `/usr/bin/env bash`, and then I won't have to restrict myself to portable POSIX shell.

Reply:
I've enabled `xtrace` when `V` is set, so that it'll be part of all of the other build output.